features/admin.userstores.v1/models/user-stores.ts

```diff
@@ -261,3 +263,13 @@ export interface UserStoreItem {
     value: string;
     disabled?: boolean;
 }
+
+/**
+ * Interface for the wizard step.
+ */
+export interface WizardStepInterface {
+    content: ReactElement;
+    icon: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
```

Comment:
Shall we get the `FunctionComponent` & `SVGProps` from the import rather than accessing them from the namespace.

This way we can ditch importing `React` when we migrate to the `jsx-runtime`.